openfisca_france/model/prestations/bourses_superieur.py

```diff
@@ -7,9 +13,247 @@ class echelon_bourse(Variable):
     label = "Echelon de la bourse perçue (de 0 à 7)"
     definition_period = MONTH
 
+    def formula(individu, period):
+        return individu('bourse_criteres_sociaux_echelon', period)
+
 
 class boursier(Variable):
     value_type = bool
     entity = Individu
     label = "Élève ou étudiant boursier"
     definition_period = MONTH
+
+    def formula(individu, period):
+        echelon = individu('bourse_criteres_sociaux_echelon', period)
+        return echelon >= 0
+
+
+class bourse_criteres_sociaux(Variable):
+    value_type = float
+    entity = Individu
+    reference = [
+        "Circulaire ESRS2013435C - Annexe 3 - Conditions de ressources et points de charge / 1 - Conditions de ressources"
+        ]
+    label = "Ressources prise en compte pour la bourse sur critères sociaux de l'enseignement supérieur"
+    definition_period = MONTH
+
+    def formula(individu, period, parameters):
+        montants = parameters(period).bourses_enseignement_superieur.criteres_sociaux.montants
+        echelon = individu('bourse_criteres_sociaux_echelon', period)
+
+        return montants.calc(echelon)
+
+
+class bourse_criteres_sociaux_eligibilite_etude(Variable):
+    value_type = bool
+    entity = Individu
+    reference = [
+        "Circulaire ESRS2013435C - Annexe 1 - Conditions d'études"
+        ]
+    label = "Satisfaction des critères d'âge pour la bourse sur critères sociaux de l'enseignement supérieur"
+    definition_period = MONTH
+
+    def formula(individu, period):
+        return individu('etudiant', period)
+
+
+class bourse_criteres_sociaux_eligibilite_age(Variable):
+    value_type = bool
+    entity = Individu
+    reference = [
+        "Circulaire ESRS2013435C - Annexe 2 - Critères d'attribution / 1 - Conditions d'âge"
+        ]
+    label = "Satisfaction des critères d'âge pour la bourse sur critères sociaux de l'enseignement supérieur"
+    definition_period = MONTH
+
+    def formula(individu, period):
+        age = individu('age', period)
+        handicap = individu('handicap', period)
+
+        return (age <= 28) + (handicap)
+
+
+class bourse_criteres_sociaux_base_ressource(Variable):
```

Comment:
Ressources multiples (et propager le nommage) :
```suggestion
class bourse_criteres_sociaux_base_ressources(Variable):
```